UnitySDK/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -4,15 +4,16 @@
 [RequireComponent(typeof(JointDriveController))] // Required to set joint forces
 public class CrawlerAgent : Agent
 {
-    [Header("Target To Walk Towards")][Space(10)]
+    [Header("Target To Walk Towards")]
+    [Space(10)]
     public Transform target;
 
     public Transform ground;
     public bool detectTargets;
     public bool respawnTargetWhenTouched;
     public float targetSpawnRadius;
 
-    [Header("Body Parts")][Space(10)] public Transform body;
+    [Header("Body Parts")] [Space(10)] public Transform body;
```

Comment:
attributes go on a separate line
https://ono.unity3d.com/unity-extra/unity-meta/files/@/ReferenceSource/CSharp/Assets/CSharpReference.cs

Reply:
and should be joined if they are short enough